# SHA1:d88f9db3bcded1a34aca7817b3f27f45dccdcd33
#
# This file is autogenerated by pip-compile-multi
# To update, run:
#
#    pip-compile-multi
#
click==7.1.2 \
    --hash=sha256:d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a \
    --hash=sha256:dacca89f4bfadd5de3d7489b7c8a566eee0d3676333fbb50030263894c38c0dc \
    # via -r base.txt, pip-tools
pip-tools==5.3.0 \
    --hash=sha256:13ab79c33778e4071500fe6390ab99b6ab8e552e40fe0fcf3e32419963f1bd5c \
    --hash=sha256:1c971e38643400ec6d0f6d528c7585d25855078c0af639806f00c4cb87455adf \
    # via -r base.txt
six==1.15.0 \
    --hash=sha256:30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259 \
    --hash=sha256:8b74bedcbbbaca38ff6d7491d76f2b06b3592611af620f8426e82dddb04a5ced \
    # via -r base.txt, pip-tools
toposort==1.5 \
    --hash=sha256:d80128b83b411d503b0cdb4a8f172998bc1d3b434b6402a349b8ebd734d51a80 \
    --hash=sha256:dba5ae845296e3bf37b042c640870ffebcdeb8cd4df45adaa01d8c5476c557dd \
    # via -r base.txt

# The following packages are considered to be unsafe in a requirements file:
pip==20.2 \
    --hash=sha256:912935eb20ea6a3b5ed5810dde9754fde5563f5ca9be44a8a6e5da806ade970b \
    --hash=sha256:d75f1fc98262dabf74656245c509213a5d0f52137e40e8f8ed5cc256ddd02923 \
    # via pip-tools
